### PR TITLE
OAEWDGT-92 - Fix sort menu + paging

### PR DESCRIPTION
**Pull request by christianvuerings**
https://jira.sakaiproject.org/browse/OAEWDGT-92

### CODE REVIEW

File: app/views/pagewidgets/_widget_list.html.erb [hunk 1]
```diff
@@ -2,11 +2,11 @@
   <div id="widget_list_sort_by">
       <label for="sort">Sort by</label>
       <select id="sort_by">
-        <option data-sortby="average_rating" data-direction="desc" selected>Highest Rated</option>
-        <option data-sortby="average_rating" data-direction="asc">Lowest Rated</option>
-        <option data-sortby="popularity" data-direction="desc">Most Popular</option>
-        <option data-sortby="title" data-direction="asc">A-Z</option>
-        <option data-sortby="title" data-direction="desc">Z-A</option>
+        <option data-sortby="average_rating" data-direction="desc" <% if (params[:s].eql? "average_rating") && (params[:d].eql? "desc") %>selected<% end %>>Highest Rated</option>
```
Comment (croby) [inline app/views/pagewidgets/_widget_list.html.erb]: Just a style thing, let's change 

``` ruby
if (params[:s].eql? "average_rating") && [...]
```

to

``` ruby
if params[:s].eql?("average_rating") && [...]
```

Comment (christianvuerings) [inline app/views/pagewidgets/_widget_list.html.erb]: Ok, that should be changed now

File: app/views/pagewidgets/_widget_list.html.erb [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
-        <option data-sortby="average_rating" data-direction="asc">Lowest Rated</option>
-        <option data-sortby="popularity" data-direction="desc">Most Popular</option>
-        <option data-sortby="title" data-direction="asc">A-Z</option>
-        <option data-sortby="title" data-direction="desc">Z-A</option>
+        <option data-sortby="average_rating" data-direction="desc" <% if params[:s].eql?("average_rating") && params[:d].eql?("desc") %>selected<% end %>>Highest Rated</option>
+        <option data-sortby="average_rating" data-direction="asc" <% if params[:s].eql?("average_rating") && params[:d].eql?("asc") %>selected<% end %>>Lowest Rated</option>
+        <option data-sortby="popularity" data-direction="desc" <% if params[:s].eql? "popularity" %>selected<% end %>>Most Popular</option>
+        <option data-sortby="title" data-direction="asc" <% if params[:s].eql?("title") && params[:d].eql?("asc") %>selected<% end %>>A-Z</option>
+        <option data-sortby="title" data-direction="desc"<% if params[:s].eql?("title") && params[:d].eql?("desc") %>selected<% end %>>Z-A</option>
```
Comment (croby) [inline app/views/pagewidgets/_widget_list.html.erb]: One more thing -- missing a space after `"desc"` here

Comment (christianvuerings) [inline app/views/pagewidgets/_widget_list.html.erb]: Sorry about that, should be fixed

**Pull request merged**